### PR TITLE
Add stateDB table name for lag and vlan

### PR DESCRIPTION
**Pull request by JipanYanga**
Signed-off-by: Jipan Yang <jipan.yang@alibaba-inc.com>

### CODE REVIEW

File: common/schema.h [hunk 1]
```diff
@@ -70,6 +70,8 @@ namespace swss {
 /***** STATE DATABASE *****/
 
 #define STATE_PORT_TABLE_NAME           "PORT_TABLE"
+#define STATE_LAG_TABLE_NAME            "PORTCHANNEL_TABLE"
```
Comment (stcheng) [inline common/schema.h]: I would suggest to change it to LAG_TABLE

Comment (JipanYanga) [reply common/schema.h]: Yes, LAG_TABLE is more aligned with APPL_DB.    How about also have "INTERFACE_TABLE" updated to "INTF_TABLE"?

It looks current python part of code changes had cfgDB taking a quite different table naming style.

**Comment by JipanYanga** [comment]
Changed to #define STATE_LAG_TABLE_NAME            "LAG_TABLE"

File: common/schema.h (hunk 1 above)
Comment (JipanYanga) [reply common/schema.h]: Changed to #define STATE_LAG_TABLE_NAME "LAG_TABLE"

**Pull request merged**